contents/docs/web-analytics/web-vs-product-analytics.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: What is the difference between web analytics and product analytics?
+sidebar: Docs
+showTitle: true
+---
+
+import { ComparisonTable } from 'components/ComparisonTable'
+import { ComparisonRow } from 'components/ComparisonTable/row'
+
+[Web analytics](/web-analytics) is a streamlined experience for tracking and monitoring your most important website metrics. [Product analytics](/product-analytics) is a more powerful and flexible experience for tracking your product.
```

Comment:
Trying out a bullet list to see if it makes it easier to skim.

Also, for the docs, we should be as concise as possible and remove any "marketing speak" e.g. "is a streamlined experienced"
```suggestion
- [Web analytics](/web-analytics) is for tracking website metrics. It's for marketing websites, blogs, or high volume B2C apps where users don't log in.
- [Product analytics](/product-analytics) for tracking your product. It's more powerful than web analytics. It's for web and mobile apps.
```